<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemDialog</class>
 <widget class="QDialog" name="itemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>707</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>D2Coding</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="inputItemlistWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>391</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="outputItemlistWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>391</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonApply">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>660</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>660</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="InputLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Input Variable(X)</string>
   </property>
  </widget>
  <widget class="QLabel" name="OutputLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>D2Coding</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Output Variable(Y)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
